docs/source/quickstart.rst

```diff
@@ -40,14 +40,17 @@ it on a toy classification dataset using SciKeras
 
     clf = KerasClassifier(
         get_model,
-        loss="sparse_categorical_crossentropy",
         hidden_layer_dim=100,
     )
 
     clf.fit(X, y)
     y_proba = clf.predict_proba(X)
 
 
+Note that SciKeras even chooses a loss function and compiles your model
+(see the :ref:`Advanced Usage` section for more details)!
```

Comment:
Suggested change: "if you want to add Keras's [loss name], specify `loss="loss_name"`".

Reply:
Thanks. I copied over the relevant section from the "Advanced" section instead:


```
To override the default loss, simply specify a loss function:

.. code-block:: diff

    -KerasClassifier(model=model_build_fn)
    +KerasClassifier(model=model_build_fn, loss="categorical_crossentropy")
```

Do you think that works?

Reply:
Isn't `categorical_crossentropy` the default, or one of the defaults? Maybe use `loss="kl_divergence"` instead?

Reply:
It's one of the unsupported cases for `loss="auto"`. But at the same time, it is relatively common in Keras. So I chose it because it's unsupported but also common.

Reply:
I added some detail.